plugins/modules/vmware_host_graphics.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Alexander Nikitin (ihumster@ihumster.ru)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_graphics
```

Comment:
```suggestion
module: vmware_host_graphics
version_added: '3.9.0'
```

We should document when this module has been added.

Let's hope the CI doesn't make it too hard for us this time :unamused: